frontend/coprs_frontend/coprs/config.py

```diff
@@ -148,7 +148,7 @@ class Config(object):
     CACHE_KEY_PREFIX = "copr_cache_"
 
     # Default value for temporary projects
-    DELETE_AFTER_DAYS = 60
+    DELETE_AFTER_DAYS = 700 #Changed from 60 to 700 at request of issue #2927
```

Comment:
Thank you for the patch.  I think you need to fix `forms.py` though - not the default value (I think default 60 days is quite OK).

Reply:
Very good catch, thanks for the response. I'll update line 559 of forms.py to reflect the change.

Reply:
+1, can you drop the first commit and rebase?

Reply:
I set the default value in config.py back to what it was before, and rounded the max value in forms.py  as requested, to be 730 days, which is 2 years. I'll drop the first 2 commits now.